plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -105,9 +105,11 @@ public void testTopNPushdown()
                 .isFullyPushedDown();
 
         // TopN over aggregation column
-        assertThat(query("SELECT sum(totalprice) AS total FROM orders GROUP BY custkey ORDER BY total DESC LIMIT 10"))
-                .ordered()
-                .isFullyPushedDown();
+        if (hasBehavior(SUPPORTS_AGGREGATION_PUSHDOWN)) {
```

Comment:
Thank you. I love how `TestingConnectorBehaviour` makes overrides unnecessary in most cases.